docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -1801,6 +1801,51 @@ paths:
       security:
       - {}
       - httpAuth: []
+  /api/statistic:
+    post:
+      tags:
+      - Statistics
+      summary: Register statistics service
+      operationId: postApiStatistic
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RegisterStatisticsRequestDto'
+        required: true
+      responses:
+        '204':
+          description: ''
+        '400':
```

Comment:
if the response doesn't carry any business value, IMO, it should be skipped and not shown in the API spec

Reply:
It's a default mapping in Tapir if the jsonBody is used as the input.